tests/agent/test_agent_server_session.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from requests import Session as RequestsSession
+
+from great_expectations.agent.agent_server_session import (
+    AgentServerSession,
+    GxAgentServerSessionConfig,
+)
+
+
+def test_server_session_session():
+    config = GxAgentServerSessionConfig(
+        gx_cloud_base_url="https://greatexpectations.io/test",
+        gx_cloud_organization_id="fd7838b2-e29e-4594-b29f-6bd378284a43",
+        gx_cloud_access_token="NGM3YTlkNGQtMWI3ZC00YTNhLTlhOTktMDQ5Y2I2YzAyNGE5",
+    )
+
+    server_session = AgentServerSession(config=config)
+
+    session = server_session.session()
+
+    assert isinstance(session, RequestsSession)
```

Comment:
`non-blocking`
This assert seems unnecessary. The type checker already enforces this behavior.

Reply:
are we running `mypy` on these tests?

Reply:
Yes, we are but I meant if the implementation itself wasn't returning the `RequestSession` we would get a `mypy` error.